source/documentation/tools.md

```diff
@@ -178,7 +179,7 @@ See: <https://github.com/RobinL/cheatsheets_etc/blob/master/jupyter_conda.md>
 
 Packrat is the most well-known package management tool for R. There's more information about it here: <https://rstudio.github.io/packrat/>
 
-It has some significant downsides. It can be quite temperamental, and difficult to debug when things go wrong - in the earlier days of the Analytical Platform, the majority of support issues related to getting Packrat working. 
+It has some significant downsides. It can be quite temperamental, and difficult to debug when things go wrong - in the earlier days of the Analytical Platform, the majority of support issues related to getting Packrat working.
```

Comment:
Perhaps also add that we (the AP team) don't have strong Packrat skills and knowledge, so they're basically on their own if they choose to use it (i.e. try as we might, they probably know more about their own problem situation than we're ever likely to).

Reply:
It's a good point about support. I've added a paragraph straight afterwards explaining we don't offer support for any of this stuff, unless it's a problem with the platform itself.